package.json

```diff
@@ -42,7 +42,7 @@
   "dependencies": {
     "ansi-to-html": "0.x",
     "antd": "2.x",
-    "appium": "1.6.x",
+    "appium": "1.6.4-beta",
```

Comment:
yeah that's a good idea. for now we'll just cut a new appium-desktop release whenever there's a new version of appium.